content/spin/contributing-docs.md

```diff
@@ -409,6 +410,16 @@ SUBCOMMANDS:
 
 Let's take a quick look at how you can use the `bart` CLI to check any content that you are wanting to contribute.
 
+### 6.7 Checking Your Content - Preview a Documentation Page on Localhost
+
+You can host the technical documentation on the localhost to view changes by using the following command: 
+
+<!-- @selectiveCpy -->
+
+```bash
+$ spin up -e "PREVIEW_MODE=1"
+```
+
```

Comment:
```suggestion

> Please note: using the `PREVIEW_MODE=1` as part of a `spin` command is safe on localhost and allows you to view content (even if the `date` setting in the content's `.md` is set to a future date). It is often the case that you will be checking content before the publishing date via your system. The developer documentation's manifest file `spin.toml` has the `PREVIEW_MODE` set to `0` i.e. `environment = { PREVIEW_MODE = "0" }`. This `spin.toml` file is correct for a production environment and should always be `0` (so that the CMS adheres to the publishing `date` setting for content on the public site). Simply put, you can use `PREVIEW_MODE=1` safely in your command line on your locahost but you should never update the `spin.toml` file (in this regard).

```